miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/MiniAppWebChromeClient.kt

```diff
@@ -38,7 +45,19 @@ internal class MiniAppWebChromeClient(
         origin: String?,
         callback: GeolocationPermissions.Callback?
     ) {
-        callback?.invoke(origin, true, false)
+        val hasCustomPermission =
+            miniAppCustomPermissionCache.readPermissions(miniAppInfo.id)
```

Comment:
It seems like we're duplicating code in several places for checking if a permission is granted (for example in `CustomPermissionBridgeDispatcher`, in `UserInfoBridgeDispatcher`, and right here).

Maybe we should add something like `isGranted(permission: MiniAppCustomPermissionType)` to `MiniAppCustomPermissionCache`? I guess this could be a refactoring improvement in another PR.

Reply:
Right, I have added something like `hasPermission(...)` to `MiniAppCustomPermissionCache` for this cleanup.
It can be seen in https://github.com/rakutentech/android-miniapp/pull/222